init.lua

```diff
@@ -72,18 +75,17 @@ function ranks.get_def(rank)
 end
 
 -- [function] Update player privileges
-function ranks.update_privs(player, trigger)
-	if type(player) == "string" then
-		player = minetest.get_player_by_name(player)
+function ranks.update_privs(name, trigger)
+	if type(name) == "string" then
+		player = minetest.get_player_by_name(name)
```

Comment:
This should be
```lua
if type(name) ~= "string" then
    name = name:get_player_name()
```
since only the string name is required below, not the entire player userdata object.

Reply:
Done!